### DISCUSSION TITLE
Apps(Point of sale) Products

### DISCUSSION THREAD

**Discussion by Alex-MRX-777** [btcpayserver/btcpayserver#4565] [Technical Support]
For point of sale, there is no possibility to create products through the API, is it possible to add this?

**Comment by bolatovumar** [comment]
@Alex-MRX-777 you can specify products through the `template` field of this API endpoint: https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Apps_PutPointOfSaleApp
